subprojects/docs/src/docs/userguide/reference/command_line_interface.adoc

```diff
@@ -64,6 +68,30 @@ $ gradle :myTask
 
 This will run the single "myTask" and also all of its <<tutorial_using_tasks.adoc#sec:task_dependencies,task dependencies>>.
 
+[[sec:disambiguate_task_options_from_built_in_options]]
+=== Specify options for a task
+
+To pass an option to a task, prefix the option name with `--` after the task name:
```

Comment:
This isn't what you're doing though.  You're adding `--` as an additional argument to divide global options from task options.  It's not being used a prefix to the option name, but as a marker to divide the 2 types of options.

"Maybe something like "The special argument '--' indicates the end of all global options. Any subsequent option arguments on the command line will be sent only to the immediately preceding task."

Reply:
Is this correct?  What happens if there are 2 tasks, `A` and `B`, which both accept argument `--profile` and I run `gradle -- A B --profile`.  Does the arg **only** go to `B` and not to `A`?

Reply:
I think you're misunderstanding -- this first section talks about how to pass options to tasks in general (information that doesn't already exist on this page), the subsection discusses using the "--" delimiter.

If this is confusing, happy to reword, but I suspect this specific bit of confusion comes from the context of this PR, not necessarily the enclosing section's text.

Reply:
Oh, I see.  I was reading the section header in the adoc and thought this was already meant to be about the disambiguation process.